connect/api/src/test/java/org/apache/kafka/connect/data/ValuesTest.java

```diff
@@ -1209,7 +1209,7 @@ protected SchemaAndValue roundTrip(Schema desiredSchema, String currentValue) {
     }
 
     protected SchemaAndValue roundTrip(Schema desiredSchema, SchemaAndValue input) {
-        String serialized = Values.convertToString(input.schema(), input.value());
+        String serialized = input != null ? Values.convertToString(input.schema(), input.value()) : null;
```

Comment:
Why we need this change? It will check `null` value in `Values.convertToString`.
https://github.com/apache/kafka/blob/6a37d5ccebaa6bbfc1d1b943f0ff761cf978da7d/connect/api/src/main/java/org/apache/kafka/connect/data/Values.java#L219

Reply:
First, the methods `input.schema()` and `input.value()` are called. The line below checks if `input` is null.

If `input` is null, calling the methods `input.schema()/value()` will result in a NPE.

Reply:
Oh, You are right, I look incorrect point